### ISSUE TITLE
Portal does not close after campaign reset

### ISSUE THREAD

**Issue by SenatorGrimm** [WarEmu/WarBugs#14652]
Destro won Fort 1, later went on to open 2 and 3. One destro WB in Fort 3 (28 total?) screwed everyone over in Fort 2 (160 people total) and closed 3 fast. Campaign resets, Server lags and kicks everyone out of Fort 2. Portal in Altdorf to Fort 2 is still open but unusable.

Expected behaviour: Don't allow multiple Forts running in parallel. Only creates problems and makes people mad for getting nothing for their efforts. Also, portal should disappear after Fort closes...

![Senatorle_020](https://user-images.githubusercontent.com/53111731/61987832-ca17ec00-b06d-11e9-89f7-a24503b915eb.jpg)

**Comment by SenatorGrimm** [comment]
I wonder whether this is part of the reason sometimes a Fort doesn't open after required Zone lock?

**Comment by Natherul** [comment]
no portals anymore and should not be an issue going forward, if it does happen again please make a new ticket.
